lib/generated/l10n.dart

```diff
@@ -2950,6 +2950,16 @@ class S {
     );
   }
 
+  /// `Archive Shared Albums`
+  String get archiveSharedAlbums {
+    return Intl.message(
+      'Archive Shared Albums',
```

Comment:
Can change this to "Archive shared albums"

Reply:
Done